test/test_grid_file.py

```diff
@@ -40,9 +35,12 @@
     GridOutCursor,
 )
 from pymongo import MongoClient
+from pymongo.database import Database
 from pymongo.errors import ConfigurationError, ServerSelectionTimeoutError
 from pymongo.message import _CursorAddress
 
+sys.path[0:0] = [""]
```

Comment:
this sys line needs to be above the `test` imports. Otherwise you can't run the test like this:
```
python test/test_grid_file.py
```